luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/factories/MapKeyValueStoreFactory.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.config.luthier.factories;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.yahoo.bard.webservice.config.luthier.Factory;
+import com.yahoo.bard.webservice.config.luthier.LuthierIndustrialPark;
+import com.yahoo.bard.webservice.data.dimension.KeyValueStore;
+import com.yahoo.bard.webservice.data.dimension.MapStore;
+
+/**
+ * A factory that is used by default to support KeyValueStore Dimensions.
```

Comment:
This documentation is incorrect. This factory doesn't build KeyValueStore Dimensions, but rather KeyValueStores.